cves/kernel/CVE-2013-6383.yml

```diff
@@ -305,9 +310,10 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: |
+    No notes.
```

Comment:
potentially your search process for this

Reply:
Elaborated on search process.